server/config/config.go

```diff
@@ -251,21 +253,29 @@ type SSLConfig struct {
 	DefaultHost      string   `yaml:"default_host" usage:"Host name to use for ACME generated cert if TLS request does not contain SNI."`
 }
 
+type ShardedRedisConfig struct {
+	Shards   []string `yaml:"shards" usage:"Ordered list of Redis shard addresses."`
+	Username string   `yaml:"username" usage:"Redis username"`
+	Password string   `yaml:"password" usage:"Redis password"`
```

Comment:
would it ever make sense for shards to have their own unique passwords/usernames? (wondering if ShardedRedisConfig could instead be replaced with a list of targets)

Reply:
The redis ring client in go-redis does not support specifying different usernames/passwords for different shards.